core/View.php

```diff
@@ -153,6 +153,11 @@ public function __construct($templateFile)
         $this->piwik_version = Version::VERSION;
         $this->userLogin = Piwik::getCurrentUserLogin();
         $this->isSuperUser = Access::getInstance()->hasSuperUserAccess();
+        // following is used in ajaxMacros called macro (showMoreHelp as passed in other templates) - requestErrorDiv
+        $this->isGeneralSettingsAdminEnabled = Controller::isGeneralSettingsAdminEnabled();
+        $this->isPluginsAdminEnabled = CorePluginsAdmin::isPluginsAdminEnabled();
```

Comment:
Those variables aren't used in the view anymore, but only below. So guess they could be local variables only.